### ISSUE TITLE
atom-terminal is activated only after triggering `atom-terminal:open-project-root` by menu

### ISSUE THREAD

**Issue by utensil** [karan/atom-terminal#63]
During using atom-terminal, I found keymap 'ctrl-shift-t' not always working. After some digging, I've located the reason for this: atom-terminal is activated only after triggering `atom-terminal:open-project-root` by menu "Open Terminal At Root", so the command of "atom-terminal:open" was not added at all.

This is caused by config item "activationCommands" in  `package.json`. In https://atom.io/docs/v1.0.0/hacking-atom-package-word-count :

> activationCommands: an Object identifying commands that trigger your package’s activation. The keys are CSS selectors, the values are Arrays of Strings identifying the command. The loading of your package is delayed until one of these events is triggered within the associated scope defined by the CSS selector.

If we remove this config item, atom-terminal would be activated at atom load time, thus fix the problem.

**Comment by utensil** [comment]
This is the reason causing #42 .

**Comment by utensil** [comment]
Latest commit 2994fc3c8048a030d2bc2e3c3e46532a8c46e063 fixes this issue by adding `atom-terminal:open` to "activationCommands", but there's no release containing the fix, that's why I encountered the problem.

**Comment by tkw1536** [comment]
Yes, @karan has yet to publish version 0.8.1. This entire issue is just a duplicate of #42. The workaround is to manually switch keybindings.

**Comment by freed00m** [comment]
Yes, please publish the 0.8.1, I want it bad.

Workaround wont work for me because there is a bug that atom editor ignores the keymap cson file. so pls :)
